docs/conf.py

```diff
@@ -57,3 +64,9 @@
 ]
 
 suppress_warnings = ["myst.xref_missing"]
+
+
+def setup(app: Sphinx) -> None:
+    from docs.utils import TransformSectionIdToName
```

Comment:
This is not needed, especially upon moving to the extension. Just add the extension to the list above and Sphinx will load it through the standard means.